cmd/dummy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
why are you throwing a dummy script in the command folder? At least through it in a separate folder called "scripts" or something of that nature